src/wallabyTest.ts

```diff
@@ -17,3 +17,4 @@ TestBed.initTestEnvironment(
     BrowserDynamicTestingModule,
     platformBrowserDynamicTesting()
 );
+TestBed.initTestEnvironment = () => {};
```

Comment:
Now my other function  throws an error: 

import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { DataErrorDisplayComponent } from './data-error-display.component';

xdescribe('DataErrorDisplayComponent', () => {
  let component: DataErrorDisplayComponent;
  let fixture: ComponentFixture<DataErrorDisplayComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [ DataErrorDisplayComponent ]
    })
    .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(DataErrorDisplayComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();   Expected undefined to be truthy.
  });
});